presto-docs/src/main/sphinx/security/authorization.rst

```diff
@@ -0,0 +1,130 @@
+=============
+Authorization
+=============
+
+Presto can be configured to enable authorization support for HTTP endpoints.
```

Comment:
Lets add another line which clarifies what this is. So something like `Authorization support will allow system administrators to control access to different HTTP endpoints in Presto`